Description: Escapes "-" symbol in manpage and use temp file in secure way
 Manpage for tabbed was using "-" symbol without escape which will
 be considered by groff as hyphen. This patch escapes the "-" in man-
 pages. This patch also replaces existing example given by upstream to embed
 an app into tabbed with a version which uses the secure temp files using
 mktemp.
 .
Author: Vasudev Kamath <kamathvasudev@gmail.com>
Forwarded: no
Last-Update: 2012-07-29
--- a/tabbed/tabbed.1
+++ b/tabbed/tabbed.1
@@ -20,7 +20,7 @@ tabbed \- generic tabbed interface
 is a simple tabbed container for applications which support XEmbed. Tabbed
 will then run the provided command with the xid of tabbed as appended
 argument. (See EXAMPLES.) The automatic spawning of the command can be
-disabled by providing the -s parameter. If no command is provided
+disabled by providing the \-s parameter. If no command is provided
 tabbed will just print its xid and run no command.
 .SH OPTIONS
 .TP
@@ -92,13 +92,13 @@ jumps to nth tab
 .B F11
 Toggle fullscreen mode.
 .SH EXAMPLES
-$ tabbed surf -e
+$ tabbed surf \-e
 .TP
-$ tabbed urxvt -embed
+$ tabbed urxvt \-embed
 .TP
-$ tabbed xterm -into
+$ tabbed xterm \-into
 .TP
-$ $(tabbed -d >/tmp/tabbed.xid); urxvt -embed $(</tmp/tabbed.xid);
+$ xid=$(tabbed \-d);urxvt \-embed $xid
 .TP
 $ tabbed -r 2 st -w '' -e tmux
 .SH CUSTOMIZATION
